GPII-1172: list of Cloud4all repositories

cstrobbe opened the pull request:
This pull request contains
- a new page (`cloud4all.html`) with a list of code repositories created by Cloud4all and that are hosted outside the GPII organisation (some even outside GitHub),
- a few changes to `index.html` (mostly descriptions of the GPII repos that were already listed),
- a `readme.md` that contains a link to http://gpii.github.io/.

cstrobbe committed:
Added Cloud4all repos

cstrobbe committed:
GPII-1172: Added CC licence and readme.md

cstrobbe committed:
GPII-1172: Firefox extension added

cstrobbe committed:
Context-related repos and RBMM rule editor added

cstrobbe commented:
Hi @sgithens 
Should the repos under https://github.com/gpii-ops also be listed?

cstrobbe commented:
Ping! Are there any comments on this pull request?

kaspermarkus commented on README.md:
```diff
@@ -0,0 +1,9 @@
+
+## Licence and Copyright
+
+Licence: Creative Commons Attribution-ShareAlike 4.0 International [CC BY-SA 4.0](LICENCE.html).
```
I think it would make sense to specify here that this page is CC, but that the rest of the repos have other licencelisted within the individual repos

kaspermarkus commented:
I think this looks great, Christophe! Thanks so much for taking this on! @colinbdclark @sgithens does this look OK to you? If so, we should go ahead and merge -- manuel was pushing for this stuff today as well

cstrobbe committed:
Added comment: referenced repos have their own lic

colinbdclark commented on LICENCE.html:
```diff
@@ -0,0 +1,236 @@
+<!-- See http://creativecommons.org/licenses/by-sa/4.0/legalcode -->
+<!-- Human-readable summary: http://creativecommons.org/licenses/by-sa/4.0/ --> 
+<head>
+  <title>Creative Commons — Attribution-ShareAlike 4.0 International — CC BY-SA 4.0</title>
```
This license has a couple of issues. First of all, we don't use copyleft licenses for the GPII, so if we need to be using a CC license, it should not be the ShareAlike version. Secondly, what does the license apply to? Just the HTML content of this Website? We should probably link to this on CC's website rather than keeping a copy in our repository.

cstrobbe replied:
Hi Colin,
The licence would only apply to the content on gpii.github.io, not to the repositories referenced in the content. I know that we don't use copyleft licences for code, but I thought CC for a document like this one wouldn't restrict the reuse of the code referenced here.
I can remove the ShareAlike to make it less restrictive. 
Should I wait a few days to give others time to comment?

colinbdclark replied:
The RtF Consortium IPR agreement specifically states the Creative Commons Attribution International license for GPII-related projects. So please go ahead and update it to be less restrictive.

cstrobbe replied:
OK, I've adapted the readme.md. I will also remove the local licence file. (I don't know if this will also remove these comments.)

colinbdclark commented on cloud4all.html:
```diff
@@ -0,0 +1,149 @@
+<html lang="en">
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="chrome=1" />
```
Why do we need this meta tag?

cstrobbe replied:
I copied this from this page that Steve Githens created: https://github.com/GPII/gpii.github.io/blob/master/index.html. 
@sgithens Do we really need that `meta` tag? According to the W3C Validator, the only correct value for the `content` attribute in that meta tag is `IE=edge`.

cstrobbe committed:
Changed licence to CC-BY 4.0

The licence is no longer ShareAlike; the link now points to https://creativecommons.org/licenses/by/4.0/legalcode instead of a file in the repo. The local licence file will be removed.

cstrobbe committed:
Local CC-BY-SA licence deleted.

(Previous commit contained a link to CC-BY on CC website.)

cstrobbe committed:
test.md deleted

The file test.md was no longer needed.

cstrobbe committed:
Meta tag for Chrome deleted.

The meta tag for Chrome was deleted. It was not clear what it was for.
(<meta http-equiv="X-UA-Compatible" content="chrome=1" />)

cstrobbe committed:
Context Aware Server now has different repo.

cstrobbe commented:
I've done some clean-up.
And the Context Aware Server now appears to live in a different GitHub repo.

javihernandez commented on cloud4all.html:
```diff
@@ -0,0 +1,151 @@
+            GNOME-Shell extension for the <abbr>GPII</abbr> Personalization Framework
+          </li>
+          <li>
+            <a href="https://build.opensuse.org/project/show/home:javihernandez:GPII:Factory">Factory repository of the <abbr>GPII</abbr> <abbr>RPM</abbr> packages</a>:
```
Hey @cstrobbe,

this is the project where last packages are living on:
https://build.opensuse.org/project/show/home:javihernandez:GPII:Factory:Bundle

javihernandez commented on cloud4all.html:
```diff
@@ -0,0 +1,151 @@
+            <a href="https://github.com/JChaconTechnosite/Cloud4ALL-Enviromental-reporter">Environmental Reporter for Android</a>.
+          </li>
+          <li>
+            <a href="https://github.com/adelga/AndroidCloud4All">Cloud4all Android implementation</a>.
```
That repo was the first implementation that FVE made for Android, and it only served as a first demo because it wasn't integrated with the GPII core-architecture framework. Do we really want to put it here?

colinbdclark replied:
Yes, it seems like this link should be changed to the [official Android repository](https://github.com/gpii/android), since FVE's features have been merged into it, right @javihernandez?

javihernandez replied:
Exactly, IMVHO, that repo doesn't add any value to this page.

cstrobbe replied:
So, to be absolutely clear, "that repo" refers to https://github.com/adelga/AndroidCloud4All? That one can be removed?

colinbdclark replied:
Correct. Along with the comment on line 148.

javihernandez commented:
It's missing a couple of personal repos,

https://github.com/javihernandez/android-user-listeners and https://github.com/javihernandez/gwc-chrome
Also, don't know if it's worth adding the SST extension for Google Chrome (https://github.com/GutiX/SSTChrome4cloud)

cstrobbe committed:
Some more repos added.

cstrobbe committed:
Linux packages; AndroidCloud4All

It is not clear whether AndroidCloud4All should be listed at all.

cstrobbe committed:
CAS is in two repos. Which one is official?

cstrobbe committed:
Unofficial CAS removed.

The CAS fork has been removed from the list; the other CAS repo now has
a proper funding acknowledgement.

cstrobbe committed:
Updated description GWC.

cstrobbe commented:
Any further comments on this pull request? We wanted this piece of documentation to be available before the review.

cstrobbe committed:
Online banking repos added.

colinbdclark commented on cloud4all.html:
```diff
@@ -0,0 +1,168 @@
+
+<body>
+  <div id="container">
+    <div class="inner">c
```
Is there a reason for this stray "c" here?

cstrobbe committed:
Typo & old AndroidCloud4all repo.

cstrobbe committed:
Comment w outdated repo removed.

cstrobbe committed:
Unclosed a elements.

javihernandez commented:
:beers:
